tests/test_tensor.py

```diff
@@ -372,3 +372,37 @@ def test_pdf_eval_2(backend):
     assert pytest.approx([-23.579605171119738], rel=5e-5) == pyhf.tensorlib.tolist(
         pdf.logpdf(pdf.config.suggested_init(), data)
     )
+
+
+def test_tensor_precision(backend):
+    tb, _ = backend
+    assert tb.float_precision in ['32b', '64b']
+    assert tb.int_precision in ['32b', '64b']
+
+
+def test_set_tensor_precision():
```

Comment:
Can this be parameterized? Or is that not any better here??

Reply:
not 100% sure it's worth it. It'll end up getting reduced massively when we merge int/float into a single precision field.